docker/pytorch/Dockerfile

```diff
@@ -149,3 +149,34 @@ RUN useradd -rm -d /home/mosaicml -s /bin/bash -u 1000 -U -s /bin/bash mosaicml
 WORKDIR /home/mosaicml
 
 USER mosaicml
+
+
+FROM pytorch_stage AS vision_stage
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG PYTORCH_VERSION
+ARG CUDA_VERSION_TAG
+
+##############################
+# MosaicML Vision Dependencies
+##############################
+
+RUN sudo apt-get update && \
+    sudo apt-get install -y --no-install-recommends \
+    # For FFCV:
+    pkg-config \
+    libturbojpeg-dev \
+    libopencv-dev \
+    # For deeplabv3:
+    ffmpeg \
+    libsm6 \
+    libxext6 && \
+    sudo apt-get autoclean && \
+    sudo apt-get clean && \
+    sudo rm -rf /var/lib/apt/lists/*
+
+RUN sudo pip3 install ffcv opencv-python numba 
+
+RUN sudo pip3 install mmcv-full==1.4.4 -f https://download.openmmlab.com/mmcv/dist/${CUDA_VERSION_TAG}/torch`echo ${PYTORCH_VERSION} | sed -E "s/([[:digit:]]).([[:digit:]]).([[:digit:]])/\1.\2.0/g"`/index.html
```

Comment:
Should we define another build arg called `MMCV_PYTORCH_VERSION`? (I appreciate this `sed`ing, but it is not very readable.) A comment here would also be helpful to explain why we need to replace version digits.

Reply:
Added MMCV_TORCH_VERSION just for pytorch_vision images.